psi4/src/psi4/libfunctional/superfunctional.cc

```diff
@@ -381,6 +384,48 @@ void SuperFunctional::add_c_functional(std::shared_ptr<Functional> fun) {
     can_edit();
     c_functionals_.push_back(fun);
 }
+void SuperFunctional::copy_density_tolerance(double cut) {
+    can_edit();
+    density_tolerance_ = cut;
+}
+void SuperFunctional::set_density_tolerance(double cut) {
+    // modify both LibXCFunctional and Functional
+    density_tolerance_ = cut;
+    if (libxc_xc_func_) {
```

Comment:
I don't think this `if` needs to be here. `x_functionals_.size()` is simply zero.

Reply:
correct, will be removed in next push.